pkg/storegateway/series_chunks.go

```diff
@@ -35,6 +41,12 @@ var (
 		// the slice with the right size.
 		New: nil,
 	})
+
+	chunkBytesSlicePool = pool.Interface(&sync.Pool{
```

Comment:
Look how we wrap this for `seriesEntrySlicePool` in the tests (in a `init()` function) so we can test the balance to ensure all slices we get from the pool are released. Then please add assertions on the balance to the existing tests where appropriate (ping me if you need guidance).

Reply:
Done, and changed the test to no longer assert on the exact number of bytes allocated.